packages/dbus.rb

```diff
@@ -3,37 +3,60 @@
 class Dbus < Package
   description 'D-Bus is a message bus system, a simple way for applications to talk to one another.'
   homepage 'https://www.freedesktop.org/wiki/Software/dbus/'
-  version '1.13.18'
+  version '1.13.18-1'
   license 'Apache-2.0'
   compatibility 'all'
   source_url 'https://dbus.freedesktop.org/releases/dbus/dbus-1.13.18.tar.xz'
   source_sha256 '8078f5c25e34ab907ce06905d969dc8ef0ccbec367e1e1707c7ecf8460f4254e'
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/dbus/1.13.18_armv7l/dbus-1.13.18-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/dbus/1.13.18_armv7l/dbus-1.13.18-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/dbus/1.13.18_i686/dbus-1.13.18-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/dbus/1.13.18_x86_64/dbus-1.13.18-chromeos-x86_64.tar.xz',
+  binary_url({
+    x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/dbus/1.13.18-1_x86_64/dbus-1.13.18-1-chromeos-x86_64.tpxz'
   })
-  binary_sha256 ({
-    aarch64: '11c36243a83f99f7a2f44194ed1e8691fc5a4dd98238a067e8fa8caf53364842',
-     armv7l: '11c36243a83f99f7a2f44194ed1e8691fc5a4dd98238a067e8fa8caf53364842',
-       i686: 'cfa8bf8ea807e3b6762a041419ffd39ce6c4888a430debb0b6a4c9aa55a63489',
-     x86_64: 'c777f379f48d88844a5f9eb1f9e38174df8645c6e76845ab0fc8f1ee58034c52',
+  binary_sha256({
+    x86_64: '3a459724ba6e7f14eacc089fb5500b20b7857aa3034cfc58edf2450ad8ac1c02'
   })
 
-  depends_on 'expat'
+  depends_on 'expat' # R
+  depends_on 'libice' # R
+  depends_on 'libsm' # R
+  depends_on 'libx11' # R
```

Comment:
We need libx11 for dbus?  :face_with_head_bandage:

Reply:
oh yeah. Check out the packages which depend upon it.